data/campaigns/Delfadors_Memoirs/scenarios/11_Wasteland.cfg

```diff
@@ -257,7 +257,7 @@
                 [/set_recruit]
                 [modify_side]
                     side=3
-                    share_maps=yes
+                    share_vision=shroud
```

Comment:
Maybe add a comment `# Delfador's side has fog=no, so share_vision=shroud is equivalent to share_vision=all`?